backend/tests/api/support/index.js

```diff
@@ -0,0 +1,144 @@
+const request = require('supertest');
+const { jwt } = require("../../helpers/strapi");
+const userFactory = require("../../user/factory");
+
+describe("Support Test", () => {
+    let authenticatedUser;
+    let adminUser;
+    beforeAll(async () => {
+        console.log("Support before all");
```

Comment:
Please delete this line. If I have this line, I forgot to delete the log. You can feel free to delete my line as well. It's good to delete unnecessary logs.

Reply:
log existing

Reply:
removed log